pkg/rest/request.go

```diff
@@ -123,7 +123,13 @@ func executeRequest(client *http.Client, request *http.Request) (Response, error
 			return Response{}, fmt.Errorf("HTTP request failed: %w", err)
 		}
 		defer func() {
-			err = resp.Body.Close()
+			closeErr := resp.Body.Close()
+			if err == nil {
```

Comment:
```suggestion
			if err == nil && closeErr != nil {
```
That's what you mean, right?

Reply:
ah... that's what I get for deciding later to wrap the close error in a nicer message 😅 The simple if err == nil worked when the code inside it was just err = closeErr

Reply:
fixed - a bit differently than suggested